src/diffusers/pipelines/deepfloyd_if/pipeline_if_superresolution.py

```diff
@@ -806,8 +806,9 @@ def __call__(
 
         # 2. Define call parameters
 
-        height = self.unet.config.sample_size
-        width = self.unet.config.sample_size
+        height, width = image.shape[2:]
+        width *= 4
+        height *= 4
```

Comment:
Can we get the *4 anywhere from the config? cc @williamberman

Reply:
hmm, I'm not sure if this is right actually